vignettes/programming_strategy.Rmd

```diff
@@ -692,7 +742,12 @@ i.e., `"lifecycle_error_deprecated"` or `"lifecycle_warning_deprecated"`,
 respectively. The unit-test should follow the corresponding format, per the
 [unit test guidance](unit_test_guidance.html#writing-unit-tests-in-admiral).
 
-### For Deprecated Functions that Issues a Warning (Phase 1)
+### For Deprecated Functions that Issues a Message (Phase 1)
+
+TBD
+
+
```

Comment:
So this section is a little tricky - so haven't filled it out.  Once we are okay with what is in admiral I will update with some advice as it was a few learnings

Reply:
section updated